src/shared/parameter/config_definitions/ssl_simulation_main_command_line_args.yaml

```diff
@@ -0,0 +1,66 @@
+- string:
+    name: interface
+    value: ""
+    description: >-
+        The interface to send and receive packets over (can be found through ifconfig)
+
+- string:
+    name: logging_dir
+    value: ""
+    description: >-
+        The directory to output logs to. Absolute paths are recommended as the working directory
+        is inside the bazel-out directory.
+
+- string:
+    name: ssl_division
+    value: "div_b"
+    options:
+        - "div_a"
+        - "div_b"
+    description: >-
+        Which ssl_division to configure the simulator to. Changes the field size and the number
+        of initial robots.
+
+- enum:
+    name: team_colour
+    enum: TeamColour
+    value: YELLOW
```

Comment:
Nit: Does it matter if we have an options field like we did for `ssl_division` (except for `BLUE` and `YELLOW`)

Reply:
Shouldn't really make a difference - the enum only allows for yellow and blue anyways